stories/assembly-aws-tech-notes.adoc

```diff
@@ -8,3 +8,4 @@ ifdef::context[:parent-context: {context}]
 {AAPonAWS} is a self-managed deployment. The following are technical notes regarding the {AAPonAWS} deployment.
 
 include::topics/tech-note-aws-add-remove-tags.adoc[leveloffset=+1]
+include::topics/tech-note-ansible-on-clouds-ops-container.adoc[leveloffset=+1]
```

Comment:
Do you need aws in file name:  tech-note-aws-ansible-on-clouds-ops-container.adoc to follow convention?

Reply:
Let us add the `aws` in there to maintain our naming convention like @chrisahl notes @prat98